CYOAfeb21

```diff
@@ -0,0 +1,225 @@
+###Standard Question Setup###
+def multich(ch1="error1", ch2="error2", ch3="error3", ch4="error4"):
+	choice= input(f"""
+	Select Option:
+		A. {ch1}
+		B. {ch2}
+		C. {ch3}
+		D. {ch4}""")
+	return choice
+	
+###Questions for naming and selecting play class.###
+def givename():
+	name= input(print("Type your name."))
+	return name
+def giveclass():
+	playclass= multich(ch1='Warrior', ch2='Wizard', ch3='Cleric', ch4='Rogue')
+	return playclass
+	#add details about stats here
+#name, position, items, attack, defense, health, magic, experience, level, playclass
+
+###Define character class###
+class Character:
+	def __init__(self, name, position, items):
+		self.name= name
+		self.position= position
+		self.items= items
+#Setters and Getters for all Characters
+#Name		
+	def name(self):
+		return self.name
+	def name(self, value):
+			self.value= value
+#Position			
+	def position(self):
+		return self.position
+	def position(self, value):
+		self.value= value
+#Items			
+	def items(self):
+		return self.items
+	def items(self, value):
+		self.value= value
+###//Define NPC Subclass//###
+###Not needed since they only need character parameters###
+
+###Start NPC Bob###
+bob= Character('Bob', 'Town', 'Main Key')
+#Function for printing bob.
+def bobsheet():
+	print(f"""
+	Name: {bob.name}
+	Position: {bob.position}
+	Items: {bob.items}""")
+#Printing Bob
+#bobsheet()	
+	
+###Define Battlers###
+class Battler(Character):
+	def __init__(self, name, position, items, attack, defense, health, magic, experience, level, playclass):
+		self.name= name
+		self.position= position
+		self.items= items
+		self.attack= attack
+		self.defense= defense
+		self.health= health
+		self.magic= magic
+		self.experience= experience
+		self.level= level
+		self.playclass= playclass
+#name, position, items, attack, defense, health, magic, experience, level, playclass
+###        ###          ###          ###         ###
+###Setters and Getters for Battlers###
+#Attack	
+	def attack(self):
+		return self.attack
+	def attack(self, value):
+		self.value= value
+#Defense	
+	def defense(self):
+		return self.defense
+	def defense(self, value):
+		self.value= value
+#Health
+	def health(self):
+		return self.health
+	def health(self, value):
+		self.value= value	
+#Magic
+	def magic(self):
+		return self.magic
+	def magic(self, value):
+		self.value= value	
+#Experience	
+	def experience(self):
+		return self.experience
+	def experience(self, value):
+		self.value= value
+#Level
+	def level(self):
+		return level.health
+	def level(self, value):
+		self.value= value	
+#Playclass
+	def playclass(self):
+		return self.playclass
+	def playclass(self, value):
+		self.value= value
+		
+###Start PC 'player'.###
+player= Battler(givename(), "Town", None, 1, 1, 1, 'none', 0, 1, giveclass())
+if player.playclass== 'A':
```

Comment:
You need this logic inside the battler class. You're actually assigning this outside of instantiation, which is frowned upon.

Reply:
You mean within the call for battler I think.

Reply:
No wair. You mean I need to tab it into battler?